test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,55 @@
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const repoName = 'jasmine-json-report';
+
+describe('Consuming GET Method GitHub API', () => {
+  it('Consume GET Service', async () => {
+    const response = await axios.get(`${urlBase}/users/aperdomob`);
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.eql('Alejandro Perdomo');
+    expect(response.data.company).to.eql('Perficient Latam');
+    expect(response.data.location).to.eql('Colombia');
+  });
+
+  it('Repo List Aperdomob', async () => {
+    const repos = await axios.get(`${urlBase}/users/aperdomob/repos`);
+    const repo = repos.data.find((e) => e.name === repoName);
+    expect(repos.data[12].name).to.eql(repo.name);
+    expect(repo.full_name).to.eql('aperdomob/jasmine-json-report');
+    expect(repo.private).to.eql(false);
+    expect(repo.description).to.eql('A Simple Jasmine JSON Report');
+  });
+
+  it('Downloading jasmine-json-report Repository', async () => {
+    const response = await axios.get(`${urlBase}/repos/aperdomob/${repoName}/zipball/master`);
+    expect(response.status).to.equal(StatusCodes.OK);
+  });
+
+  it('Checking Name, Path and Sha in README file', async () => {
+    const response = await axios.get(`${urlBase}/repos/aperdomob/${repoName}/contents`);
+    const nameFile = 'README.md';
+    const pathFile = 'README.md';
+    const shaFile = '360eee6c223cee31e2a59632a2bb9e710a52cdc0';
+
+    const readmeFile = response.data.find((e) => e.name === nameFile);
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(readmeFile).to.containSubset({
+      name: nameFile,
+      path: pathFile,
+      sha: shaFile
+    });
+  });
+
+  it('Checking MD5 in README file', async () => {
+    const response = await axios.get('https://raw.githubusercontent.com/aperdomob/jasmine-json-report/master/README.md');
```

Comment:
Great job!
I would only change something, the hypermedia for example for this Readme is available in the response from previous calls, so, create a variable and assigned it when you receive the correspondent URL.

I'm not sure if this is the only one, if it isn't i would suggest to change them all

Reply:
as @manuelq12 says, you can take advantage of previous calls to get the URL.

For example the URL to this call can be get from the readmeFile (of previous test), since it has a property called `download_url`
![image](https://user-images.githubusercontent.com/10209180/176712427-fb4923f8-1b80-4c81-9044-9ecb93347581.png)

Reply:
you have two options:
- you can declare as global variable (that means, declare it outside of the describes) and use let, instead of var.
- use nested describes, in that way the variable will be within the scope.